docs/tutorials/enable-totp-openstack.md

```diff
@@ -0,0 +1,80 @@
+# Configuring Time-Based One-Time Password (TOTP) on OpenStack
+
+Author: Ramon Grullon
+
+## Introduction
+
+Horizon added time-based one-time password (TOTP) authentication support,
+leveraging the already existing two factor authentication from Keystone.
+Now, if a user activates TOTP on Keystone, it gets activated on Horizon too.
+
+## Prerequisites
+
+- A working OpenStack 2023.2 environment deployed using kolla-ansible.
+- Enabled TOTP on all control nodes
+- Admin access to the OpenStack dashboard and CLI.
+- Access to the kolla-ansible configuration files.
+
```

Comment:
Consider adding a section here, something like "### Reconfigure Keystone", explaining TOTP is not enabled by default and link to https://docs.openstack.org/keystone/latest/admin/auth-totp.html#configuring-totp, which explains the configuration needed to adjust Keystone. I can see a user following this guide and missing the keystone configuration step.

Reply:
In v2 of the doc, I'll be creating a seperate section on how to configure keystone for TOTP.